webapi/appsettings.json

```diff
@@ -161,8 +161,8 @@
   "DocumentMemory": {
     "GlobalDocumentCollectionName": "global-documents",
     "ChatDocumentCollectionNamePrefix": "chat-documents-",
-    "DocumentLineSplitMaxTokens": 30,
-    "DocumentParagraphSplitMaxLines": 100,
+    "DocumentLineSplitMaxTokens": 64,
+    "DocumentParagraphSplitMaxLines": 512,
```

Comment:
I think it's reasonable to change the defaults as you've done here. This of course depends a lot on the document that your feeding in so it would also help to add some more documentation, maybe just in the comments in this file, about the fact that these properties can be used to tune things.

I also think it could be useful to expose these parameters as part of the document import API so that they could change per document. We won't do that work now but we could create an item on the backlog and fit it in where it makes sense.

Reply:
Agree.  I think might want to consider how to drive these requirements into the semantic-memory since chunking logic will reside there.  (Currently, there is no way to specify the token threshold for semantic-memory.)

https://github.com/microsoft/chat-copilot/issues/180